### PR TITLE
Deriv x update navigation

### PR DESCRIPTION
**Pull request by alamin-deriv**
Changes:

-   Added Deriv X on the Navigation
- Added Deriv X on the Mobile view

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/elements/off-canvas-menu.js [hunk 1]
```diff
@@ -283,6 +220,85 @@ export const OffCanvasMenuWrapper = (props) => {
...
+                        <Flex mb="2rem">
+                            <NavCard
+                                aria_label="Derivx"
+                                icon={() => <img src={DerivX} alt="Derivx" width="32" height="32" />}
+                                content={
+                                    <Localize translate_text="CFD trading on a customisable multi-asset platform." />
+                                }
+                                title={<Localize translate_text="Deriv X" />}
+                                onClick={handleArrowClick}
+                                to={derivx_url}
```
Comment (ShuvoHabib) [inline src/components/elements/off-canvas-menu.js:250]: Let's use relative url for this. ie: '/derivx'

File: src/common/constants.js [hunk 2]
```diff
@@ -60,11 +60,12 @@ export const p2p_playstore_url = 'https://play.google.com/store/apps/details?id=
 export const pushwoosh_app_code = 'DD293-35A19'
 export const sample_rate = 25
 export const zoho_url = 'https://deriv.zohorecruit.com'
+export const derivx_url = 'https://deriv.com/derivx/'
```
Comment (ShuvoHabib) [inline src/common/constants.js:63]: We can user relative url in the component instead of this.

Comment (mustofa-binary) [reply src/common/constants.js:63]: please take note of the domain as well 🙏  `${domain.name}`, can be either (.com|.me)

File: src/components/custom/other-platforms.js [hunk 3]
```diff
@@ -146,6 +156,20 @@ export const DMT5Card = ({ is_selected, is_ppc_redirect, word_break_cover }) =>
     </StyledLink>
 )
 
+export const DerivXCard = ({ is_selected, is_ppc_redirect, word_break_cover }) => (
+    <StyledLink ariaLabel="Deriv X" to={is_ppc_redirect ? 'URL' : '/derivx'}>
```
Comment (mustofa-binary) [inline src/components/custom/other-platforms.js:160]: where does 'URL' link goes to?

File: src/components/custom/other-platforms.js [hunk 4]
```diff
@@ -207,6 +231,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             {exclude.toLowerCase() !== 'dtrader' && <TraderCard />}
             {exclude.toLowerCase() !== 'dbot' && <BotCard />}
             {exclude.toLowerCase() !== 'dmt5' && <DMT5Card is_ppc_redirect={is_ppc_redirect} />}
+            {exclude.toLowerCase() !== 'Derivx' && <DerivXCard is_ppc_redirect={is_ppc_redirect} />}
```
Comment (mustofa-binary) [inline src/components/custom/other-platforms.js:234]:
```suggestion
            {exclude.toLowerCase() !== 'derivx' && <DerivXCard is_ppc_redirect={is_ppc_redirect} />}
```

**Review by mustofa-binary — CHANGES_REQUESTED** [review]
please help to check suggested changes

File: src/components/custom/other-platforms.js [hunk 5]
```diff
@@ -207,6 +231,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             {exclude.toLowerCase() !== 'dtrader' && <TraderCard />}
             {exclude.toLowerCase() !== 'dbot' && <BotCard />}
             {exclude.toLowerCase() !== 'dmt5' && <DMT5Card is_ppc_redirect={is_ppc_redirect} />}
+            {exclude.toLowerCase() !== 'derivx' && <DerivXCard is_ppc_redirect={is_ppc_redirect} />}
```
Comment (sean-binary) [inline src/components/custom/other-platforms.js:234]:
```suggestion
            {exclude.toLowerCase() !== 'derivx' && <DerivXCard />}
```

Comment (sean-binary) [reply src/components/custom/other-platforms.js:234]: I don't think you will need the `is_ppc_redirect` prop here as the landing page will not be nested under `/landing/*`.

File: src/components/custom/other-platforms.js [hunk 6]
```diff
@@ -452,9 +492,7 @@ export const NavResources = ({ onClick }) => (
             icon={() => <img src={Community} alt="community" width="24" height="24" />}
             title={localize('Community')}
             onClick={onClick}
-            to=""
-            is_community_link
-            external="true"
```
Comment (sean-binary) [inline src/components/custom/other-platforms.js:457]: Any reason these were changed?

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**